tests/unit/amo/components/TestGuidesAddonCard.js

```diff
@@ -0,0 +1,248 @@
+import * as React from 'react';
+
+import GuidesAddonCard, {
+  GuidesAddonCardBase,
+} from 'amo/components/GuidesAddonCard';
+import AddonTitle from 'amo/components/AddonTitle';
+import AMInstallButton from 'core/components/AMInstallButton';
+import AddonCompatibilityError from 'amo/components/AddonCompatibilityError';
+import {
+  dispatchClientMetadata,
+  fakeAddon,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import {
+  FATAL_ERROR,
+  INCOMPATIBLE_NOT_FIREFOX,
+  INCOMPATIBLE_UNDER_MIN_VERSION,
+  INSTALLING,
+  UNKNOWN,
+} from 'core/constants';
+import { setInstallError, setInstallState } from 'core/actions/installations';
+import { getErrorMessage } from 'core/utils/addons';
+import { createInternalAddon, loadAddonResults } from 'core/reducers/addons';
+
+describe(__filename, () => {
+  const _loadAddonResults = (addon = createInternalAddon(fakeAddon), store) => {
```

Comment:
This should be `addon = fakeAddon` because the reducer is responsible for calling `createInternalAddon()`.